app/views/users/new.html.erb

```diff
@@ -0,0 +1,55 @@
+<h1>Create new user</h1>
+
+<!-- <form action="/users" method="post" accept-charset="UTF-8">
+  <label for="username">Username:</label>
+  <input type="text" id="username" name="user[username]"><br><br>
```

Comment:
- [ ] The commented-out forms should use `users_path` for its helper action. ([screenshot](https://gitlab.com/microverse/guides/projects/requirements_screenshots/raw/master/images/rails/forms/user_form1.png) [screenshot](https://gitlab.com/microverse/guides/projects/requirements_screenshots/raw/master/images/rails/forms/user_form2.png) from [Odin](https://www.theodinproject.com/courses/ruby-on-rails/lessons/forms))